libs/test-utils/src/matchers.ts

```diff
@@ -0,0 +1,5 @@
+export function buttonPressEventMatcher():
+  | React.TouchEvent<HTMLButtonElement>
+  | React.MouseEvent<HTMLButtonElement> {
+  return expect.objectContaining({ detail: expect.any(Number) });
+}
```

Comment:
Could see an argument for putting this in `libs/ui`, closer to where it's defined

Reply:
No strong opinion - this is fine with me